src/composables/assetDetails.ts

```diff
@@ -8,23 +13,31 @@ interface ISharedAssetDetails {
   tokenBalance?: ITokenBalance;
   tokens?: IToken[];
   isMultisig?: boolean;
+  transactionList?: ICommonTransaction[];
 }
 
-const assetDetails = ref<ISharedAssetDetails | null>(null);
-
-const sharedAssetDetails = computed((): ISharedAssetDetails => assetDetails.value || {});
+const sharedAssetDetails = reactive<ISharedAssetDetails>({});
```

Comment:
In this case `reactive` makes more sense because when using this we are always accessing the `sharedAssetDetails` properties. So by uing the `reactive` we are able to write:
```
sharedAssetDetails.contractId
```
instead of
```
sharedAssetDetails.value.contractId
```
instea

Reply:
Makes sense